### PR TITLE
Added a scenario for C3 integration with multi ksqldb and connect

### PR DESCRIPTION
**Pull request by arodoni**

### CODE REVIEW

**Review by rohit2b — CHANGES_REQUESTED** [review]
Added some changes, apart from that looks great!

File: controlcenter-multi-ksql-connect/README.rst [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+==================================
+Set the current tutorial directory
+==================================
+
+Set the tutorial directory for this tutorial under the directory you downloaded
+the tutorial files:
+
+::
+   
+  export TUTORIAL_HOME=<Tutorial directory>/secure-authn-encrypt-deploy
```
[the remark this answers is not in the document]
Comment (arodoni) [reply controlcenter-multi-ksql-connect/README.rst:43]: Fixed

File: controlcenter-multi-ksql-connect/README.rst [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+The assumption is that you’ve set up Early Access credentials following `the
+instruction
+<https://github.com/confluentinc/operator-earlyaccess/blob/master/README.rst>`__.
+
+#. Install Confluent Operator using Helm:
+
+   ::
+
+     helm upgrade --install operator confluentinc/confluent-operator --set image.registry=confluent-docker-internal-early-access-operator-2.jfrog.io
```
[the remark this answers is not in the document]
Comment (arodoni) [reply controlcenter-multi-ksql-connect/README.rst:57]: Done

File: controlcenter-multi-ksql-connect/README.rst [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+   ::
+
+     openssl genrsa -out $TUTORIAL_HOME/ca-key.pem 2048
+    
+   ::
+
+     openssl req -new -key $TUTORIAL_HOME/ca-key.pem -x509 \
+       -days 1000 \
+       -out $TUTORIAL_HOME/ca.pem \
+       -subj "/C=US/ST=CA/L=MountainView/O=Confluent/OU=Opeator/CN=TestCA"
```
[the remark this answers is not in the document]
Comment (arodoni) [reply controlcenter-multi-ksql-connect/README.rst:100]: Fixed

**Review by rohit2b — APPROVED** [review]
LGTM!

**Pull request merged**